include/libXbSymbolDatabase.h

```diff
@@ -61,7 +61,9 @@ extern "C" {
 #define Lib_XACTENG  "XACTENG"
 #define Sec_XACTENG  Lib_XACTENG
 #define Lib_XAPILIB  "XAPILIB"
+#define Sec_XID      ".XID"
 #define Sec_XPP      "XPP"
+#define Sec_XPPDat   ".XPP&Dat"
```

Comment:
If I consult an ASCII chart, I see that "." comes first than "X", so .XID and .XPP are probably in the wrong order